### ISSUE TITLE
add tags for each note

### ISSUE THREAD

**Issue by TheKiteRunning** [glfharris/ankidown#5]
Hi, is it possible to import tags for each note like :

> [If you want to add tag1 and tag2 to every line you’re importing, add the following to the top of the text file: tags:tag1 tag2](https://apps.ankiweb.net/docs/manual.html#adding-tags)

**Comment by glfharris** [comment]
Yeah, at the moment it will use the key "Tags" to populate the tags field for each note.
